File: apps/s-2-admin/src/api/attendee/AttendeeWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { StringFilter } from "../../util/StringFilter";
import { StringNullableFilter } from "../../util/StringNullableFilter";
import { BookingWhereUniqueInput } from "../booking/BookingWhereUniqueInput";

export type AttendeeWhereInput = {
  id?: IntFilter;
  email?: StringFilter;
  name?: StringFilter;
  timeZone?: StringFilter;
  locale?: StringNullableFilter;
  booking?: BookingWhereUniqueInput;
};

File: apps/cactus-demo/src/APIs/Attendee/Dtos/AttendeeWhereInput.cs
namespace CactusDemo.APIs.Dtos;

public class AttendeeWhereInput
{
    public int? Id { get; set; }

    public string? Email { get; set; }

    public string? Name { get; set; }

    public string? TimeZone { get; set; }

    public string? Locale { get; set; }

    public int? Booking { get; set; }
}
